src/webview/js/__tests__/rewriteHtml.js

```diff
@@ -1,10 +1,10 @@
 /** @jest-environment jsdom-global */
 // @flow strict-local
 
-import rewriteImageUrls from '../rewriteImageUrls';
+import rewriteHTML from '../rewriteHtml';
```

Comment:
Something's not quite right about this line :wink:

Reply:
I can't figure out what's wrong! :P

Reply:
Ah, well spotted, Greg! I see it too—check the variable against the path, letter-by-letter. 🙂

Reply:
Oh, got it now! Capitalization!